discovery_engine_core/core/src/engine.rs

```diff
@@ -302,60 +298,66 @@ where
     /// Updates the stacks with data related to the top key phrases of the current data.
     ///
     /// Requires a threshold below which new items will be requested for a stack.
-    async fn update_stacks(
-        &mut self,
-        history: &[HistoricDocument],
-        request_new: usize,
-    ) -> Result<(), Error> {
+    #[instrument(skip(self))]
```

Comment:
this makes `just test` somewhat noisy again, should we also skip the other arguments of the method?

Reply:
We can avoid `instrument` completely. We will add this in another PR and see what we can do for the log from dart.